cmd/haproxy-cfg-generator/cfg_generator.go

```diff
@@ -28,11 +28,14 @@ import (
 )
 
 const (
-	ARKTOS_API_PORT      = 8080
-	MAX_CONN_PER_BACKEND = 500000
-	CONNECTION_TIMEOUT   = "10m"
-	PROXY_PORT           = 8888
-	TEMPLATE_FILE        = "haproxy.cfg.template"
+	ARKTOS_API_SECURE_PORT = 443
+	ARKTOS_API_PORT        = 8080
```

Comment:
we should call this ARKTOS_API_INSECURE_PORT

Reply:
good one. it is the previous CONST NAME, but I think it will be good to rename it to match the newly added secure-port CONST.

Reply:
fixed.